alacritty/src/display/mod.rs

```diff
@@ -367,6 +370,15 @@ pub struct Display {
     meter: Meter,
 }
 
+#[derive(Debug, Default, Copy, Clone)]
```

Comment:
```suggestion
/// Pending renderer updates.
///
/// All renderer updates are cached to be applied just before rendering, to avoid platform-specific rendering issues.
#[derive(Debug, Default, Copy, Clone)]
```